CONTRIBUTING.md

```diff
@@ -21,23 +21,23 @@ The following is a set of guidelines for contributing to Atala PRISM KMM SDK. Th
 
 ## What should I know before I get started?
 
-### Atala PRISM
+### Identus
 
-Atala PRISM is a self-sovereign identity (SSI) platform and service suite for verifiable data and digital identity. Built on Cardano, it offers the core infrastructure for issuing DIDs (Decentralized identifiers) and verifiable credentials alongside tools and frameworks to help expand your ecosystem.
+Identus is a self-sovereign identity (SSI) platform and service suite for verifiable data and digital identity. Built on Cardano, it offers the core infrastructure for issuing DIDs (Decentralized identifiers) and verifiable credentials alongside tools and frameworks to help expand your ecosystem.
 The complete platform is separated into multiple repositories:
 
 * [atala-prism-building-blocks](https://github.com/hyperledger-labs/open-enterprise-agent) - Repo that contains the servers Building Blocks.
 * [atala-prism-wallet-sdk-swift](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift) - Repo for the Swift version of the SDK.
 * [atala-prism-wallet-sdk-ts](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts) - Repo for the TS version of the SDK.
 * [atala-prism-mediator](https://github.com/input-output-hk/atala-prism-mediator) - Repo for DIDComm V2 Mediator
 
-### Atala PRISM KMM SDK
+### Identus KMM SDK
 
-Atala PRISM KMM software development kit will help adoption within Kotlin platforms (Java/Android) by providing key functionalities. For more information about the SDK, please have a look at the [Readme](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/blob/main/README.md)
+Identus KMM software development kit will help adoption within Kotlin platforms (Java/Android) by providing key functionalities. For more information about the SDK, please have a look at the [Readme](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/blob/main/README.md)
 
 ### Your First Code Contribution
 
-Unsure where to begin contributing to Atala PRISM KMM SDK? You can start by looking through the [Readme](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/blob/main/README.md) that provides all the steps to setup your environment.
+Unsure where to begin contributing to Identus KMM SDK? You can start by looking through the [Readme](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/blob/main/README.md) that provides all the steps to setup your environment.
```

Comment:
`Identus KMM SDK` should be `Edge Agent SDK KMM` according to the [replacement table](https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit#gid=0)

Reply:
Updated in the main branch.